media-libs/libmad/libmad-0.16.4.recipe

```diff
@@ -3,13 +3,14 @@ DESCRIPTION="MAD is a high-quality MPEG audio decoder. It currently supports \
 MPEG-1 and the MPEG-2 extension to lower sampling frequencies, as well as the \
 de facto MPEG 2.5 format. All three audio layers -- Layer I, Layer II, and \
 Layer III (i.e. MP3) -- are fully implemented."
-HOMEPAGE="http://mad.sourceforge.net/"
-COPYRIGHT="2000-2004 Underbit Technologies, Inc."
+HOMEPAGE="https://codeberg.org/tenacityteam/libmad"
+COPYRIGHT="2000-2004 Underbit Technologies, Inc.
+	2021-2024 Tenacity Team"
 LICENSE="GNU GPL v2"
-REVISION="8"
-SOURCE_URI="ftp://ftp.mars.org/pub/mpeg/libmad-0.15.1b.tar.gz"
-CHECKSUM_SHA256="bbfac3ed6bfbc2823d3775ebb931087371e142bb0e9bb1bee51a76a6e0078690"
-PATCHES="libmad-$portVersion.patchset"
+REVISION="1"
+SOURCE_URI="https://codeberg.org/tenacityteam/libmad/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="f4eb229452252600ce48f3c2704c9e6d97b789f81e31c37b0c67dd66f445ea35"
+SOURCE_DIR="$portName"
```

Comment:
Didn't check this but iirc this will fail on 32bit secondaryArchitecture where $portName resolves into libmad_x86 then.

Reply:
Checked, should be:

```suggestion
SOURCE_DIR="libmad"
```